src/main/java/seedu/address/model/timingrecommender/TimingRecommender.java

```diff
@@ -65,11 +64,10 @@ private void addParticipants(Collection<ContactIndex> participantIndices) {
                 .filter(Optional::isPresent)
                 .map(Optional::get)
                 .forEach(this.participants::add);
-        // Append timetables for each participant
-        // Wait for @kennycjy to handle his Module Tag + Lesson integration
-        // planned implementation:
-        // for each participant : (1) build (2) add their timetable
-        // to be handled in a separate method.
+
+        this.participants.stream()
+                .map(Person::getTimetable)
+                .forEach(this.schedules::add);
```

Comment:
At this point, the user's own timetable is not added in yet right?

Reply:
It is added in through the `MeetCommandParser`.

```Java
public MeetCommand parse(String args) throws ParseException {
        ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(args);
        List<String> indexArray = Arrays.stream(argumentMultimap.getPreamble().split(" "))
                .filter(x -> !x.isEmpty())
                .collect(Collectors.toList());

        Set<ContactIndex> indices =
                ParserUtil.parseIndices(indexArray);

        indices.add(new ContactIndex(0));

        return new MeetCommand(indices, locationHashSet);
    }
```